tests/test_transform.py

```diff
@@ -141,7 +139,7 @@ def test_cora_transforms_pck(self):
         print("Found %d cora pck scenarios" % len(test_scenarios))
 
         for scenario_filename in test_scenarios:
-
+            self.maxDiff = None
```

Comment:
Sorry, I added this in to give more debug info when we were comparing the test outputs but it can be taken out now.